observatory-platform/observatory/platform/api/app/test_query.py

```diff
@@ -0,0 +1,194 @@
+import unittest
+from observatory.platform.api.app.api.query import (create_es_connection,
+                                                              create_search_body,
+                                                              validate_dates,
+                                                              process_response,
+                                                              create_schema,
+                                                              parse_args,
+                                                              search)
+
+from observatory.platform.api.app.server import app
+
+
+class TestQuery(unittest.TestCase):
```

Comment:
Similar to the tests, structuring the methods in such a way that enables multiple versions of the API to have tests written. Imagining a scenario in say 6 months time, where these initial tests will still be required (as we are still supporting the current version) but there may also be two other versions also running.